goaviatrix/controller.go

```diff
@@ -422,3 +430,54 @@ func (c *Client) SetControllerVpcDnsServer(enabled bool) error {
 		Action: action,
 	}, BasicCheck)
 }
+
+func ValidateBackupConfig(d *schema.ResourceData) error {
```

Comment:
This function should not be in `goaviatrix` package. This is specific to the resource schema and should be in the controller config resource file.